### PR TITLE
Add Js API docs for all of our new/missing docs

### PR DESCRIPTION
**Pull request by jsundra**
Adds docs for new/upgraded:
`random-preview`
`assets-preview`
`system.login`/`system.logout`
`system.cursor`
`system.camera`
`this.particles`

### CODE REVIEW

File: source/includes/assets-preview.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+```javascript
+const assets = require('assets-preview');
+```
+
+The Assets API can be used to fetch `AssetData` based on Ids, Names, and Tags.
+
+## Methods
+
+```javascript
+// Get all Assets available to this experience.
```
Comment (thegoldenmule) [inline source/includes/assets-preview.md]: The convention we've been following with slate is _generally_ to pull comments like this out into:

```
> Get all assets available...

` ` `
var all = assets.all();
` ` `
```

It looks nicer...

File: source/includes/assets-preview.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+var aquatic = assets.byTag('Aquatic')
+log.info('Aquatic Assets: ' + aquatic.length);
+
+var seashells = assets.query('SeaShell');
+log.info('Seashell Assets: ' + seashells.length);
+```
+
+The returned `AssetData` has the following properties:
+
+<b>Guid</b> - The unique Id of this Asset.
```
Comment (thegoldenmule) [inline source/includes/assets-preview.md]: `<b>` ? Why not **?

File: source/includes/system.md [hunk 3]
```diff
@@ -151,6 +151,25 @@ system.experiences.edit(a.id);
 
 Experiences may be retrieved in a variety of ways.
 
+## Login
+
+> Check if the user is logged in.
+```javascript
```
Comment (thegoldenmule) [inline source/includes/system.md]: Have you tried rendering this? _Almost_ positive a new line is required between the > and back ticks. Also: this is what I was referring to in the other comment about pulling comments into > remarks.

File: source/includes/random-preview.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Random (Preview)
```
[the remark this answers is not in the document]
Comment (jsundra) [reply source/includes/random-preview.md]: :'( Well that's embarrassing

File: source/includes/system.md (hunk 3 above)
Comment (jsundra) [reply source/includes/system.md]: You're right - forgot to add the tweak to the commit before pushing. Will update!

File: source/includes/assets-preview.md (hunk 2 above)
Comment (jsundra) [reply source/includes/assets-preview.md]: No reason, just forgot `**` was a thing. I'll change it, it looks better

File: source/includes/random-preview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gdamron) [reply source/includes/random-preview.md]: You might be able to blame me for that one. I added it to my tentative release notes after digging through a couple months of commits. It looked like it hadn't reached master yet :\

File: source/includes/particles.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+}
+```
+
+The Particles API is available on any Element that contains an Asset with an underlying ParticleSystem.
+
+## Emission
+
+```javascript
+// Emit 10 particles.
+this.particles.emit(10);
```
Comment (gdamron) [inline source/includes/particles.md]: use ">" style comments here as well for consistency?

**Pull request merged**